gist.el

```diff
@@ -186,7 +194,7 @@ With a prefix argument, makes a private paste."
 
 (defun gist-created-callback (gist)
   (let ((location (oref gist :html-url)))
-    (gist-list-reload t)
+    (gist-list-reload "" t)
```

Comment:
somehow I don't like too much this "magic" empty-string username. Passing nil would be more consistent with optional arguments (here, and for other occurrences in the diff).